pkg/core/xds/sockets.go

```diff
@@ -17,6 +17,11 @@ func MetricsHijackerSocketName(tmpDir, name, mesh string) string {
 	return socketName(filepath.Join(tmpDir, fmt.Sprintf("kuma-mh-%s-%s", name, mesh)))
 }
 
+// MeshMetricsDynamicConfigurationSocketName generates a socket path that will fit the Unix socket path limitation of 104 chars
+func MeshMetricsDynamicConfigurationSocketName(tmpDir string) string {
```

Comment:
are we sure we don't need this per mesh?

Reply:
I don't think so, DPP can be only part of one mesh, also this is not scoped by mesh, and we create only one socket per dpp for this communication